app/controllers/providers/legal_aid_applications_controller.rb

```diff
@@ -1,11 +1,17 @@
 module Providers
   class LegalAidApplicationsController < ProviderBaseController
+    include Pagy::Backend
     legal_aid_application_not_required!
 
+    DEFAULT_PAGE_SIZE = 20
+
     # GET /provider/applications
     def index
-      # TODO: Add pagination at some point
-      @applications = current_provider.legal_aid_applications.latest.limit(10)
+      @pagy, @legal_aid_applications = pagy(
+        current_provider.legal_aid_applications.latest,
+        items: params.fetch(:page_size, DEFAULT_PAGE_SIZE),
```

Comment:
will `params[: page_size]` ever be present?

Reply:
It's very difficult to test the functionality without it. For example, to test that the pagination links look like this:

![pagy-start-sequence](https://user-images.githubusercontent.com/213040/55628670-36ae6880-57a9-11e9-853f-510482340ca8.png)

You'd have to create something in the order of 160 applications.

Reply:
ah ok that makes sense